apps/web/src/components/mobileComponents/MobileConnectWallets.vue

```diff
@@ -0,0 +1,151 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import useWallet from '@/composables/wallet'
+import Carousel from '@/components/Carousel.vue'
+import Slide from '@/components/Slide.vue'
+import VueFeather from 'vue-feather'
+import TermsOfService from '@/components/TermsOfService.vue'
+const  {
+  activeWallets,
+  walletProviderAddresses,
+  selectAddress,
+  selectProvider,
+} = useWallet()
+
+const authFlowCardNumber = ref(1)
+
+const selectedProivder = ref(null as null | string)
+
+const termsCheckbox = ref(true)
+const showTermsOfService = ref(false)
+const convertString = (inputString: string) => {
```

Comment:
Rename to something like `abbreviateAddress`. This is also probably used several places throughout the app; should we have a `useHelpers` composable or something like that?